projects/aca-content/src/lib/components/info-drawer/library-metadata-tab/library-metadata-form.component.ts

```diff
@@ -124,7 +140,12 @@ export class LibraryMetadataFormComponent implements OnInit, OnChanges, OnDestro
 
   ngOnInit() {
     this.updateForm(this.node);
-
+    this.form.controls.title.statusChanges
+      .pipe(takeUntil(this.onDestroy$))
+      .subscribe(
+        () =>
+          (this._titleErrorTranslationKey = this.form.controls.title.errors?.empty ? 'LIBRARY.ERRORS.ONLY_SPACES' : 'LIBRARY.ERRORS.TITLE_TOO_LONG')
```

Comment:
Do we have to show the 'Title too long' error even if there is no content on the field? Right now, if the user removes the title of the library completely, then the application is also reporting to use 256 or less characters in the title.
Addl note: This is not the behaviour on the create library dialog

Reply:
Yeah naming of that variable was a bit confusing but error itself is correct in case you did not type any value as it says "use 256 or less characters". Should I align these errors with exactly the same like for create library? @swapnil-verma-gl

Reply:
I guess its ok. The error does make sense as you already pointed out. It just seemed a bit weird when I first saw it. We can leave it as is for now

Reply:
as discussed I renamed translation key

Reply:
as discussed I renamed translation key

Reply:
Yes, looks much better now, thanks!!